src/DetailsView/components/adhoc-static-test-view.tsx

```diff
@@ -45,6 +45,16 @@ export const AdhocStaticTestView = NamedFC<AdhocStaticTestViewProps>(
         );
         const displayableData = props.configuration.displayableData;
 
+        const stepsText = (): string => {
```

Comment:
This looks almost identical to the `stepsText` function in `details-list-issues-view.tsx`. We could pull it into the ` fastPassProvider` and have it take the selected test as a parameter and then call it in both places to cut down on duplicate code.

Reply:
Done!